run_udf_externally.json

```diff
@@ -52,7 +52,7 @@
         }
     ],
     "returns": {
-        "description": "The data processed by the UDF service. Returns a raster data cube if a raster data cube was passed for `data`. If an array was passed for `data`, the returned value is defined by the context and is exactly what the UDF returned.",
+        "description": "The data processed by the UDF service.\n\n* Returns a raster data cube, if a raster data cube was passed for `data`. Details on the dimensions and dimension properties (name, type, labels, reference system and resolution) depend on the UDF.\n* If an array was passed for `data`, the returned value can be of any data type, but is exactly what the UDF returned.",
```

Comment:
> if a raster data cube was passed for `data`

if a raster data cube is passed for `data`

> If an array was passed for `data`, the returned value can be of any data type, but is exactly what the UDF returned.

If an array is passed for `data`, the returned value can be of any data type, but is exactly what the UDF returns.